src/Errors.ts

```diff
@@ -13,15 +13,17 @@ export class AppStoreError extends Error {
 
   errorCode: number
   isRetryable: boolean
+  retryAfter?: number
 
   // https://developer.apple.com/documentation/appstoreserverapi/ratelimitexceedederror
   isRateLimitExceeded: boolean
 
-  constructor(errorCode: number, errorMessage: string) {
+  constructor(errorCode: number, errorMessage: string, { retryAfter}:{ retryAfter?: number } = {}) {
```

Comment:
The last parameter looks a little clunky. Why not just use `retryAfter?: number`?
```typescript
constructor(errorCode: number, errorMessage: string, retryAfter?: number) {...}
```

Reply:
Sure!